lib/checks/aria/required-children.js

```diff
@@ -119,7 +129,8 @@ this.data(missing);
 // Only review empty nodes when a node is both empty and does not have an aria-owns relationship
 if (
 	reviewEmpty.includes(role) &&
-	node.children.length === 0 &&
+	!hasContentVirtual(virtualNode) &&
```

Comment:
I think this should have `ignoreAria: true`, otherwise I think this will be picked up:

```html
<div role="list" aria-label="My list"><span></span></div>
```